### ISSUE TITLE
Mongoose - Cannot Read Property 'toJSON' of null

### ISSUE THREAD

**Issue by wdhodges** [strapi/strapi#5129]
**Describe the bug**
When running 

**Steps to reproduce the behavior**
1. Navigate to `localhost:1337/admin/plugins`
2. You will get the error in terminal that is listed below
3. Click in the `Search for a file...` in the top right, enter some text and then clear it out
4. List of uploads will appear

**Expected behavior**
Load a list of current uploads without workaound

**Code snippets**
```
[2020-01-29T16:41:37.340Z] error TypeError: Cannot read property 'toJSON' of null
    at refToStrapiRef (C:\dev\project-orion\serenity\node_modules\strapi-connector-mongoose\lib\mount-models.js:232:34)
    at C:\dev\project-orion\serenity\node_modules\strapi-connector-mongoose\lib\mount-models.js:270:26
    at Array.map (<anonymous>)
    at C:\dev\project-orion\serenity\node_modules\strapi-connector-mongoose\lib\mount-models.js:269:75
    at Array.forEach (<anonymous>)
    at transform (C:\dev\project-orion\serenity\node_modules\strapi-connector-mongoose\lib\mount-models.js:253:27)
    at model.Document.$toObject (C:\dev\project-orion\serenity\node_modules\mongoose\lib\document.js:2922:21)
    at model.Document.toObject (C:\dev\project-orion\serenity\node_modules\mongoose\lib\document.js:3076:15)
    at C:\dev\project-orion\serenity\node_modules\strapi-connector-mongoose\lib\queries.js:419:46
    at Array.map (<anonymous>)
    at C:\dev\project-orion\serenity\node_modules\strapi-connector-mongoose\lib\queries.js:419:15
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async Object.find (C:\dev\project-orion\serenity\node_modules\strapi-plugin-upload\controllers\Upload.js:146:18)
    at async C:\dev\project-orion\serenity\node_modules\strapi\lib\middlewares\router\utils\routerChecker.js:83:22
    at async module.exports (C:\dev\project-orion\serenity\node_modules\strapi-plugin-users-permissions\config\policies\permissions.js:40:14)
    at async C:\dev\project-orion\serenity\node_modules\strapi-utils\lib\policy.js:118:7
```

**System**
- Node.js version: 13.5
- NPM version: 6.13.6
- Strapi version: v3.0-beta.18.6
- Database: MongoDB
- Operating system: Windows 10 v1903

**Comment by wdhodges** [comment]
I was looking at the error, and then the source. I think on the `strapi-connector-mongoose\lib\mount-models.js:232:34`, the `ref.JSON` should be `ref.JSON()`. I could be totally wrong, but just trying to help.

**Comment by JelmerV-WFC** [comment]
Got the same error when I navigate to `/admin/plugins/upload` after the migrate to beta-18.6.

**Comment by JelmerV-WFC** [comment]
@wdhodges fixed by #5137

**Comment by wdhodges** [comment]
Thank you! @JelmerV-WFC
